src/lib.rs

```diff
@@ -66,14 +75,15 @@ impl LoggerUi {
             if ui.button("Clear").clicked() {
                 *logs = vec![];
             }
-
-            egui::ComboBox::from_label("Log Level")
-                .selected_text(self.loglevel.to_string())
-                .show_ui(ui, |ui| {
-                    ui.selectable_value(&mut self.loglevel, log::Level::Info, "Info");
-                    ui.selectable_value(&mut self.loglevel, log::Level::Warn, "Warn");
-                    ui.selectable_value(&mut self.loglevel, log::Level::Error, "Error");
-                });
+            ui.label("Log Level");
+            // I didn't find elegant ways to iterate an Enum :(
+            for level in LEVELS
+            {
+                if ui.selectable_label( self.loglevels[level as usize - 1], level.as_str()).clicked()
+                {
+                    self.loglevels[level as usize - 1] = !self.loglevels[level as usize - 1];
+                }
+            }
```

Comment:
```suggestion
            ui.menu_button("Log Levels", |ui| {
              for level in LEVELS
              {
                  if ui.selectable_label( self.loglevels[level as usize - 1], level.as_str()).clicked()
                  {
                      self.loglevels[level as usize - 1] = !self.loglevels[level as usize - 1];
                  }
              }
            }
```

Please replace the label with a menu_button containing the selectable Levels